api/odilo.py

```diff
@@ -191,8 +191,8 @@ def record_info_to_metadata(cls, book, availability):
 
         title = book.get('title')
         subtitle = book.get('subtitle')
-        series = book.get('series')
-        series_position = book.get('seriesPosition')
+        series = book.get('series').strip() or None
+        series_position = book.get('seriesPosition').strip() or None
```

Comment:
This is totally random -- I was asked to investigate some Odilo problems, and the problems were much bigger than this -- to the point that there's currently no solution -- but these are some relatively simple problems with incoming data that I was able to solve.